tasks/eth/021-holocene-protocol-versions/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
```

Comment:
let's remove this and use `OWNER_SAFE` where we load the address, (e.g. in `SingleSign`)